### PR TITLE
Add --match option to batch command to limit scope of processed plotdefs

### PR DESCRIPTION
**Pull request by iand**

### CODE REVIEW

**Comment by dennis-tra** [comment]
That's a QoL improvement!

File: batch.go [hunk 1]
```diff
@@ -228,16 +236,25 @@ func (p *ProcessingProfile) processPlotDefs(ctx context.Context, cfg *PlotConfig
...
 	} else {
 		infs = os.DirFS(filepath.Dir(p.Source))
-		fnames = []string{filepath.Base(p.Source)}
+		matchGlob = filepath.Base(p.Source)
+		// fnames = []string{filepath.Base(p.Source)}
+	}
+	if cfg.MatchGlob != "" {
+		fnames, err = fs.Glob(infs, cfg.MatchGlob)
+	} else {
+		fnames, err = fs.Glob(infs, matchGlob)
```
Comment (dennis-tra) [inline batch.go:254]: Could put this `matchGlob` into the value of the `StringFlag` option above. Then you could skip this if statement here and always just use `cfg.MatchGlob`

**Review by dennis-tra — APPROVED** [review]

File: batch.go (hunk 1 above)
Comment (iand) [reply batch.go:254]: I don't think so, matchGlob is different when the source is a single file (it is set to the filename)

Comment (dennis-tra) [reply batch.go:254]: You're right; apologies for the noise!

**Pull request merged**